Cargo.toml

```diff
@@ -14,11 +14,6 @@ default-members = ["crates/revm"]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
-[profile.release]
-lto = true
-codegen-units = 1
```

Comment:
I would leave `lto` and `codegen-units` as they can help with performance.

Reply:
remnants from debugging